BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Channel/TransmissionTest.cs

```diff
@@ -423,9 +423,7 @@ public async Task SendAsyncLogsIngestionReponseTimeEventCounter()
                         Assert.AreEqual("IngestionEndpoint-ResponseTimeMsec", payload["Name"].ToString());
                         Assert.IsTrue((int)payload["Count"] >= 5);
                         // Max should be more than 30 ms, as we introduced a delay of 30ms in SendAsync.
-#if NETCOREAPP2_1
-                        Assert.IsTrue((float)payload["Max"] >= 30);
-#elif NETCOREAPP3_1
+#if NETCOREAPP
```

Comment:
If we are planning to cover .NET 5.0, .NET 6.0+, then better to include `NET` along with `NETCOREAPP`. Ref:  https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation

Reply:
`NET` and `NETCOREAPP` both cover the same frameworks.

In my opinion, `NET` sounds ambiguous between the older NetFrameworks (4.5 - 4.8).
I would prefer to use `NETCOREAPP` to differentiate.

Reply:
since NETCOREAPP covers NET5.0 and higher, this is fine i think.